tls/s2n_server_new_session_ticket.c

```diff
@@ -244,6 +232,8 @@ S2N_RESULT s2n_tls13_server_nst_write(struct s2n_connection *conn, struct s2n_st
     RESULT_ENSURE_REF(conn);
     RESULT_ENSURE_REF(output);
 
+    struct s2n_ticket_fields *ticket_fields = &conn->tls13_ticket_fields;
+
```

Comment:
Nit: Add a `RESULT_ENSURE_REF(ticket_fields);` after this initialization?

Reply:
It can't be null. We just set it to the address of a valid structure.